src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3724,7 +3724,10 @@ class LanguageClauseSegment(BaseSegment):
 
     type = "language_clause"
 
-    match_grammar = Sequence("LANGUAGE", Ref("ParameterNameSegment"))
+    match_grammar = Sequence(
+        "LANGUAGE",
+        OneOf(Ref("ParameterNameSegment"), Ref("SingleQuotedIdentifierSegment")),
```

Comment:
I wonder if it's worth changing `ParameterNameSegment` to `NakedIdentifierSegment` here so that they're consistently flagged as `identifier` in the parse tree?

Reply:
Good shout. Done in https://github.com/sqlfluff/sqlfluff/pull/3068/commits/af33652b9013a0451cddb03565c5270b5be9e130

Reply:
NOt sure if you want to clean them in this PR's scope - but approved 👍